misago/users/management/commands/removeips.py

```diff
@@ -0,0 +1,14 @@
+
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+
+UserModel = get_user_model()
+
+class Command(BaseCommand):
+    help = "Deletes all user registered IPs"
+
+    def handle(self, *args, **options):
+      for user in UserModel.objects.all():
```

Comment:
`objects.all()` will attempt to load all users in database to operational memory. If you have more than few dozens of users, you will end with OutOfMemory error.